es-vue-base/src/lib-components/EsReview.vue

```diff
@@ -165,11 +269,33 @@ export default {
         },
         /**
          * Review Comment Truncation Limit
+         * Also indicates whether to show the modal view of component
          */
         commentLimit: {
             default: 225,
             type: [Number, Boolean],
         },
+        /**
+         * Developer Response
+         * Expected structure:
+         * developer (Number),
+         * response_text (String),
+         * developer_logo (String),
+         * modified (Date)
+         */
+        response: {
+            required: false,
+            type: [Number, String, Date, null],
+            default: null,
+        },
```

Comment:
This should be defined as `type: Object` and then have an expected shape. However, I know it'll be ugly but we should flatten this into 4 different props. Having an prop be an object has caused us lots of headaches in the past.